src/components/ToggleSwitch.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { ChangeEvent } from "react";
+import { Theme } from "../theme";
+
+const Label = styled.label`
+  display: flex;
+  align-items: center;
+  gap: 10px;
```

Comment:
For future reference. For consistency, we'll use `rem` over `px` to match the other components. I've handled updating these is a separate branch.